cmd/webhook/main.go

```diff
@@ -68,7 +76,7 @@ func newDefaultingAdmissionController(ctx context.Context, cmw configmap.Watcher
 	return defaulting.NewAdmissionController(ctx,
 
 		// Name of the resource webhook.
-		"webhook.pipeline.tekton.dev",
+		os.Getenv("MUTATING_WEBHOOK_NAME"),
```

Comment:
looks like `defaultEnv` is back but unused atm.  If you want to use that here and on the modified lines below then +1  otherwise remove `defaultEnv` and I think this is ready to lgtm!  Thanks and sorry for all the delays @jbarrick-mesosphere